pkg/api/roles.go

```diff
@@ -244,10 +244,25 @@ func (hs *HTTPServer) declareFixedRoles() error {
 		Grants: []string{string(models.ROLE_ADMIN)},
 	}
 
+	annotationsReaderRole := accesscontrol.RoleRegistration{
+		Role: accesscontrol.RoleDTO{
+			Name:        "fixed:annotations:reader",
+			DisplayName: "Annotation reader",
+			Description: "Read annotations and tags",
+			Group:       "Annotations",
+			Version:     1,
+			Permissions: []accesscontrol.Permission{
+				{Action: accesscontrol.ActionAnnotationsRead, Scope: accesscontrol.ScopeAnnotationsAll},
+				{Action: accesscontrol.ActionAnnotationsTagsRead, Scope: accesscontrol.ScopeAnnotationsAll},
```

Comment:
Tags are stored in a separate table in the DB (they're also stored explicitly in the annotation table, but I'm not sure why), so I think we can scope the tags by tag ID instead of scoping them by annotation ID.

Reply:
Ah, I see that you've already added a scope for it (`ScopeAnnotationsTagsAll`), so this might just be a typo :)